app/src/main/res/navigation/main_nav.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/main_nav"
+    app:startDestination="@id/splashFragment">
+
+    <fragment
+        android:id="@+id/comingSoonFragment"
+        android:name="com.grappus.android.basemvi.fragments.ComingSoonFragment"
+        android:label="ComingSoonFragment"
```

Comment:
Can we also move these to `strings.xml` with `translatable="false"` attribute please. This is the base project, the people following this will implement exactly the same.